text/0000-informational-commands.md

```diff
@@ -0,0 +1,766 @@
+tool yarn-deduplicate / yarn-deduplicate@v1.1.1
+```
+
+</details>
+
+### `volta list <package>`
+
+List all fetched versions of a specific package, along with its associated binaries.
+
+#### Human
+
+The basic format is:
+
+```sh
+volta list <package> --print=human
+
+    <version> [(default|current @ <project path>)]
+        binaries: [<binary name>]...
+        platform:
+            runtime: node@<version>
+            packager: built-in npm|<npm|yarn>@<version>
+```
+
+For the TypeScript config specified in the canonical example:
+
+```sh
+volta list package typescript --print=human
+
+    v3.4.5
+        binaries: tsc, tsserver
+        platform:
+            runtime: node@v12.2.0
+            packager: built-in npm
+
+    v3.0.3 (default)
+        binaries: tsc, tsserver
+        platform:
+            runtime: node@v12.2.0
+            packager: built-in npm
+```
+
+#### Plain
+
+The basic format is:
+
+```sh
+volta list <package> --print=plain
+tool <tool> / <package>@<version> node@<version> <npm|yarn>@<built-in|version> [(default|current @ <path>)]
+```
+
+For the TypeScript config specified in the canonical example:
+
+```sh
+volta list typescript --print=plain
+tool tsc / typescript@v3.4.5 node@12.2.0 npm@built-in
+tool tsserver / typescript@v3.4.5 node@12.2.0 npm@built-in
+tool tsc / typescript@v3.0.3 node@12.2.0 npm@built-in (default)
+tool tsserver / typescript@v3.0.3 node@12.2.0 npm@built-in (default)
+```
+
+### `volta list <tool>`
+
+#### Human
+
+The basic format is:
+
+```sh
+volta list <tool> --print=human
+⚡️ tool <tool> available from:
+
+    <package>@<version> [(default|current @ <project path>)]
+        platform:
+            runtime: node@<version>
+            packager: built-in npm|<npm|yarn>@<version>
+```
+
+For the TypeScript config specified in the canonical example:
+
+```sh
+volta list tsc --print=human
+⚡️ tool tsc available from:
+
+    typescript@v3.4.5
+        platform:
+            runtime: node@v12.2.0
+            packager: built-in npm
+
+    typescript@v3.0.3 (default)
+        platform:
+            runtime: node@v12.2.0
+            packager: built-in npm
+```
+
+#### Plain
+
+The basic format is:
+
+```sh
+volta list <tool> --print=plain
+tool <tool> / <package>@<version> node@<version> <npm|yarn>@<built-in|version> [(default|current @ <path>)]
+```
+
+For the TypeScript config specified in the canonical example:
+
+```sh
+volta list tsc
+tool tsc / typescript@v3.4.5 node@12.2.0 npm@built-in
+tool tsc / typescript@v3.0.3 node@12.2.0 npm@built-in (default)
+```
+
+## Deprecating `volta current`
+
+Since `volta list` subsumes (and substantially extends upon) the functionality of `volta current`, `volta current` should be deprecated when `volta list` is implemented. The deprecation should include a warning that the command will be removed in a future version and information about how to use `volta list --current <tool>` as a replacement.
```

Comment:
I like this approach a lot, it allows us to introduce the command and the deprecation warning without a breaking change, and then the next time we have a breaking change we can take that opportunity to remove the `current` command (that I suspect no one is using anyway given its limited functionality).